addons/purchase_stock/tests/test_reordering_rule.py

```diff
@@ -1175,3 +1175,41 @@ def test_supplierinfo_last_purchase_date(self):
         replenishment_info = self.env['stock.replenishment.info'].create({'orderpoint_id': orderpoint.id})
         supplier_info = replenishment_info.supplierinfo_ids
         self.assertEqual(supplier_info.last_purchase_date, dt.today().date(), "The last_purhchase_date should be set to the most recent date_order from the purchase orders")
+
+    def test_reordering_rule_multicurrency(self):
+        """
+            trigger a reordering rule in foreign currency
+        """
+        eur = self.env.ref('base.EUR')
```

Comment:
I'm not sure you can count on the company currency always being USD, but maybe it's only a problem in other versions (there have been issues with it in nightly no-demo tests for 17)